app/code/local/Excellence/User/controllers/IndexController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+class Excellence_User_IndexController extends Mage_Core_Controller_Front_Action
+{
+ public function deleteAction()
+  {   
+   if( $this->getRequest()->getParam('user_id') > 0 )
+    {
+     $user_id = $this->getRequest()->getParam('user_id');
```

Comment:
create a function in model file called deleteUser and put all this code there. 

controller should always have as much less code as possible. this is very important, all keep as less code in controller as possoble.